README.md

```diff
@@ -1,7 +1,6 @@
 Terraform Cloud/Enterprise Go Client
 ==============================
 
-[![Build Status](https://circleci.com/gh/hashicorp/go-tfe.svg?style=shield)](https://circleci.com/gh/hashicorp/go-tfe)
```

Comment:
What do you think about adding a [Github Actions build badge](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge) instead?